includes/class-give-readme-parser.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Give Readme Parser
+ *
+ * @package     Give
+ * @subpackage  Admin/Readme_Parser
+ * @copyright   Copyright (c) 2018, WordImpress
+ * @license     https://opensource.org/licenses/gpl-license GNU Public License
+ * @since       2.1.4
+ */
+class Give_Readme_Parser{
+	/**
+	 * Readme file url
+	 *
+	 * @since  2.1.4
+	 * @access private
+	 * @var
+	 */
+	private $file_url;
+
+	/**
+	 * Readme file content
+	 *
+	 * @since  2.1.4
+	 * @access private
+	 * @var
+	 */
+	private $file_content;
+
+	/**
+	 * Give_Readme_Parser constructor.
+	 *
+	 * @param string $file_url
+	 */
+	function __construct( $file_url ) {
+		$this->file_url = $file_url;
+		$this->file_content = ! empty( $this->file_url ) ? file_get_contents( $this->file_url ) : '';
```

Comment:
@ravinderk 

In WordPress, we have a specific function to retrieve the content from URL 

We should used that in place of Core PHP functions